CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [v1.4.0] - 2023-12-08
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.3.10...v1.4.0)
+
+### Changed
+* Enable support for ABAC Conditional Relationships by default. ([#1220](https://github.com/openfga/openfga/pull/1220))
```

Comment:
We are not calling it ABAC in the docs, just "Conditional Relationships"

Reply:
ABAC Conditions + Conditional Relationship Tuples

Those are the terms we're using

Reply:
https://github.com/openfga/openfga.dev/pull/583